pkg/storage/streams/store.go

```diff
@@ -60,16 +64,32 @@ type Store interface {
 
 // streamStore is a store for streams
 type streamStore struct {
-	segments    segments.Store
-	segmentSize int64
+	segments     segments.Store
+	segmentSize  int64
+	rootKey      []byte
```

Comment:
Is t his different from `eestream.GenericKey`?

Reply:
Yes. `rootKey` is like a password. It is used to derive the `eestream.GenericKey/Key` used for the actual encryption, but `rootKey` itself is not used to encrypt data.